pkg/metricscollector/prommetrics.go

```diff
@@ -121,6 +121,16 @@ var (
 		},
 		[]string{"namespace", "type", "resource"},
 	)
+
```

Comment:
also, please move this code next to the code where we set activity, so we have similar stuff together in the code